### ISSUE TITLE
Proposal: rename or alias `compose()` to `blend()`

### ISSUE THREAD

**Issue by koresar** [stampit-org/stamp-specification#115]
Proposal: **use more accurate and a less confusing word for the `compose()` function.**

The core of this specification is the `compose()` function. However, it is
* slightly related to the **object** composition, and
* not at all related to the **function** composition.

Almost every person out there confuses what that `compose()` function actually is. I have a dozen of examples. As well as many face to face discussions.

What the `compose()` function actually does is it **merges the descriptors** using the standardized algorithm. We could rename the function `merge()` instead. However... We can't use this word. In JavaScript it means object (deep/shallow) merging.

But the word **BLEND** is unique to software development.

Pros:
* The `blend()` word sounds exactly what the `compose()` function is actually doing - blends stamps.
* There will be much less confusion among software dev gurus, and less argument out in the Internets.
* The `blend()` is shorter. :)))))))))))))

Cons:
* This specification would need to be released as v2.0.0 since it's a breaking change.

**Comment by FredyC** [comment]
Does it really need to be a breaking change? I mean you can add the alias now and keep good old `compose` there, maybe even with some deprecation warning if you like. I wouldn't certainly advice to just over the hoop and change it.

Personally, I don't mind `compose` and I don't like much `blend`, but whatever :)

**Comment by koresar** [comment]
Yeah. Alias looks like a better idea.

**Comment by ericelliott** [comment]
I think this is a bad idea.

1. `compose()` actually does do object composition to produce the descriptors
2. `compose()` actually does set up function composition (which is exactly what happens when init functions are applied)
3. NOBODY has ever expressed confusion about this to me the entire time we've been using `compose()`.
4. API changes should not be taken lightly -- especially when they're potentially breaking changes.
5. Aliases suck because people are not sure which name is the "right" name to use. It adds a decision point that doesn't need to be there.

In short, if it ain't broke, don't fix it.

**Comment by koresar** [comment]
All valid points. Closing.
